apps/console/src/components/pages/protected/overview/DashboardSuggestedActions.tsx

```diff
@@ -0,0 +1,69 @@
+import { Card, CardContent, CardTitle } from '@repo/ui/cardpanel'
+import { Lock, ArrowUpFromLine, UserRoundPlus } from 'lucide-react'
+import CreatePolicyUploadDialog from '@/components/pages/protected/policies/create/form/create-policy-upload-dialog.tsx'
+import React from 'react'
+import { useRouter } from 'next/navigation'
+
+const DashboardSuggestedActions = () => {
+  const router = useRouter()
+
+  const handleViewProfile = () => {
+    router.push('/user-settings/profile')
+  }
```

Comment:
This should to to org settings I think 
```suggestion
  const handleSecureOrganization = () => {
    router.push('/organization-settings/authentication')
  }
```